components/Calender/index.vue

```diff
@@ -23,22 +23,26 @@
 </template>
   
 <script setup>
+import {getDaysOfMonth, getDaysOfPrevMonth} from "../../util/getDate";
```

Comment:
create helper function in `utils` folder so that [nuxt 3 - utils](https://nuxt.com/docs/guide/directory-structure/utils) will auto import your functions